com.unity.shadergraph/Editor/Data/Attributes/SRPFilterAttribute.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace UnityEditor.ShaderGraph
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    public class SRPFilterAttribute : ContextFilterableAttribute
```

Comment:
Is this intended to be public?

Reply:
Same as previous comment, will make internal